### ISSUE TITLE
WS issue after TLS termination

### ISSUE THREAD

**Issue by wey-gu** [vesoft-inc/nebula-studio#599]
**Describe the bug (__must be provided__)**

When studio is behind HTTPS/WSS, it'll got WebSocket ERROR.

**Your Environments (__must be provided__)**

* Studio-version: 3.7.0

**How To Reproduce(__must be provided__)**

With this docker compose yaml and nginx.conf, we could easily reproduced the issue.

```yaml
version: '3.4'
services:
  web:
    image: vesoft/nebula-graph-studio:v3.7.0
    environment:
      USER: root
    ports:
      - 7001
    networks:
      - nebula-net
  nginx:
    image: nginx:1.17.6
    ports:
      - 7001:443
    volumes:
      - ./nginx.conf:/etc/nginx/nginx.conf
      - ./cert:/cert
    networks:
      - nebula-net

networks:
  nebula-net:
    external: true
```

nginx.conf:
```conf
events {
}

http {
    server {
        listen 443 ssl;
        server_name nebula-demo.siwei.io;
        ssl_certificate /cert/fullchain.pem;
        ssl_certificate_key /cert/privkey.pem;
        location / {
            proxy_pass http://web:7001;
        }

        location /nebula_ws {
            proxy_pass http://web:7001;
            proxy_http_version 1.1;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "upgrade";
        }
    }
}
```

**Expected behavior**

working

**Additional context**

Screen Captured as follow:

https://github.com/vesoft-inc/nebula-studio/assets/1651790/3198b58a-7e20-4c95-af9e-43802a8cfa67

**Comment by wey-gu** [comment]
cc @hetao92 :-D

**Comment by sbocahu** [comment]
we reproduce the same issue with nginx-ingress controller with tls termination in front of a nebule-studio helm-deployed instance on kubernetes

any workaround or fix you know of ?

**Comment by salahaz** [comment]
@wey-gu any updates on this issue ?

**Comment by wey-gu** [comment]
After checking with @huaxiabuluo to know it's already fixed in 3.8.0 release, and I verified to confirm it's fixed :)

**Comment by wey-gu** [comment]
3.8.0 over wss in container

https://github.com/vesoft-inc/nebula-studio/assets/1651790/1b81652e-d7be-448c-a369-d739bf786ce2

**Comment by huaxiabuluo** [comment]
@salahaz @sbocahu 
Under the HTTPS protocol, the WebSocket service link prefix needs to be changed from `ws://` to `wss://`, this bug has been fixed in version 3.8, please upgrade to the latest version.

**Comment by salahaz** [comment]
Thank you very much @wey-gu  and @huaxiabuluo for your support!!! It is resolved.
